### DISCUSSION TITLE
simbot3.0机器人发送群消息后，能否撤回该消息

### DISCUSSION THREAD

**Discussion by SamsaraEventually** [simple-robot/Discussions#7] [Q&A] [answered]
我使用Group发送群消息后也拿到了回执，里面实现的删除方法调用后一直是返回false，是我这边的方法使用问题吗？

**Comment by ForteScarlet** [comment, marked as answer]
版本号以及具体的相关代码是怎样地

**Reply by SamsaraEventually** [reply]
public void onGroupMsgAdminTwelve(GroupMessageEvent event) {
        Bot bot = event.getBot();
        Group source = event.getSource();

        MessageReceipt messageReceipt = source.sendBlocking("iii");

        messageReceipt.deleteIfSupportBlocking();
    }
代码是这样，使用的是 simboot-core-spring-boot-starter 3.0.0.preview.8.0 和 simboot-core-spring-boot-starter 3.0.0.preview.8.0

**Reply by ForteScarlet** [reply]
你应该尝试直接使用 `deleteBlocking` ，并且此版本已经比较陈旧了，尝试适当地更新版本，比如使用 `3.0.0-beta.2` 的核心与mirai `v3.0.0.0-beta-M3` 的mirai组件 ，
或者 `v3.0.0-beta.3` 的核心和即将发布的 `v3.0.0.0-beta.1` 的mirai组件

_**注意处理preview到beta更新过程中存在的不兼容变更**_

如果在更新之后仍然无法撤回，可以将此问题反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues)

**Reply by SamsaraEventually** [reply]
谢谢，问题已经解决了，是因为旧版本的问题

**Comment by Kvi121** [comment]
group.sendBlocking
怎么引用回复内容呢？
